### PR TITLE
Add 1.16 Enhancemment Exceptions

### PR DESCRIPTION
**Pull request by kacole2**
Adding the 1.16 Enhancement Exceptions

Signed-off-by: Kendrick Coleman <kendrickc@vmware.com>

### CODE REVIEW

File: releases/release-1.16/exceptions.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Exceptions to Code Freeze requested in 1.16
+# Google Group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+# Release Team Lead: Lachlan Evenson ([@lachie83](https://github.com/lachie83))
+
+- name: add status code and content check for HTTP Probe
```
Comment (jeefy) [inline releases/release-1.16/exceptions.yaml]: Nittest of Nits: Add should be capitalized to match the Issue title.

Comment (kacole2) [reply releases/release-1.16/exceptions.yaml]: that is nit picky... just rebased.

**Pull request merged**